main.css

```diff
@@ -0,0 +1,413 @@
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
+}
+@keyframes zoomy {
+  from{
+    transform: scale(1);
+  }
+  to{
+    transform: scale(1.1);
+  }
+}
+/* move figcaption down from picture and increase font size*/
+figcaption {
+  padding-top: 10px;
+  font-size: 20px;
+  text-align: center;
+  color: rgb(255, 0, 0);
+}
+
+/* footer */
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 10vh;
+  color: white;
+}
+
+/* game page layout */
+/* turn the whole game page into a flexbox to align items */
+#game-page {
+  display: flex;
+  justify-content: center;
+  height: 60vh;
+  color: rgb(0, 117, 25);
+}
+
+/* NAV BAR ON GAMEPAGES */
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  height: 5vh;
+  font-size: 30px;
+  padding: 15px;
+  background-color: rgb(49, 27, 70);
+  border: darkslateblue 5px solid;
+  border-radius: 25px;
+  width: 70vw;
+  margin: auto;
+  margin-bottom: 20px;
+}
+/* unified color of nav links and added a little hover animation */
+a:link {
+  text-decoration: none;
+  color: rgb(200, 111, 228);
+}
+
+a:visited {
+  color: rgb(200, 111, 228);
+}
+/* give to zoom animation to the nav bar links as well*/
+a:hover {
+  animation-name: zoomy;
+  animation-duration: 2s;
+}
+/* how to layout */
+/* assign flex size of 1 to tell screen how much size to take up */
+#how-to {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  flex: 1;
+  text-align: center;
+  margin-left: 5vw;
+  align-items: center;
+  height: 50vh;
+  background-color: rgb(245, 241, 37);
+  border: 5px green solid;
+  border-radius: 20px;
+}
+
+#how-to-title {
+  color: rgb(0, 117, 25);
+}
+/* made the section for determining which cyclops the player is a grid for layout*/
+#which-cyclops {
+  display: inline-grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  width: 250px;
+  animation-name: spinny;
+}
+/* targeted images directly after paragraphs to make the images in which cyclops smaller */
+p ~ img {
+  width: 50px;
+  height: 65px;
+  padding-left: 20px;
+}
+
+/* make the images rotate 360 degrees when hovered over just for fun */
+p ~ img:hover{
+  animation-name: spinny;
+  animation-duration: 1s;
+}
+
+@keyframes spinny {
+  from{
+    transform: rotate(0deg);
+  }
+  to{
+    transform: rotate(360deg);
+  }
+}
+
+section p {
+  display: inline;
+  font-size: 20px;
+  color: rgb(0, 0, 0);
+}
+/* added iframe dimensions to the css to control easier */
+iframe {
+  width: 250px;
+  height: 150px;
+}
+
+/* game board layout */
+/* added flex of 3 to display this section 3 times larger than the how to section */
+#game-board {
+  flex: 3;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* tictactoe construction */
+
+#tictactoe {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  padding: 10px;
+}
+
+#left-top {
+  width: 7.5vw;
+  height: 7.5vw;
```

Comment:
You can cut down some code by adding a class inside these divs, ie:
```
<div class="board-square" id="left-top"></div>
<div class="board-square" id="mid-top"></div>
```
and then set prop in the css file as 
```
.board-square{
  width: 7.5vw;
  height: 7.5vw;
}
```
and this will apply to all the squares with the class property added